src/main/java/io/opentraffic/reporter/HttpClient.java

```diff
@@ -73,6 +74,12 @@ public static String DO(HttpEntityEnclosingRequestBase request) {
     CloseableHttpResponse response = null;
     String v = null;
     try {
+      //set some timeouts
+      RequestConfig.Builder builder = RequestConfig.custom();
+      builder.setConnectTimeout(1000);
+      builder.setConnectionRequestTimeout(1000);
+      builder.setSocketTimeout(10000);
+      request.setConfig(builder.build());
```

Comment:
i also added timeouts to requests to the python map matching so it doesnt wait forever but shows an error instead